modules/python/types/source/types.i

```diff
@@ -47,3 +47,93 @@
 
 %template(VectorRowColInt) std::vector<types::RowCol<sys::SSize_T> >;
 %template(VectorSizeT) std::vector<size_t>;
+
+// Pickle utilities
+%pythoncode
+%{
+    import cPickle as pickle
+%}
+
+%extend types::RowCol<double>
```

Comment:
I assume there's no way to do this extension on the original templated class so that we don't have this repeated code for each type..?